### PR TITLE
add dubbo 2.6.5 & 2.7.0 related links

### PR DESCRIPTION
**Pull request by techyang**
## What is the purpose of the change

add 2.6.5 & 2.7.0 ralated links

## Brief changelog
incubator-dubbo-website/blog/zh-cn/download.md
incubator-dubbo-website/blog/en-us/download.md

### CODE REVIEW

File: blog/en-us/download.md [hunk 1]
```diff
@@ -5,7 +5,17 @@
 
 > Release Notes: https://github.com/apache/incubator-dubbo/releases
 
-### 2.6.4 (latest version)
+### 2.7.0 (latest version)
```
Comment (ralf0131) [inline blog/en-us/download.md]: 2.7.0 is not released yet.

And the link should be updated too.

Once a release is performed, the final links should be something like this:

https://dist.apache.org/repos/dist/release/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-source-release.zip

The following link is just a temporary link just for the vote purpose.

https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-source-release.zip

**Comment by techyang** [comment]
OK, I'll do it later.

File: blog/en-us/download.md [hunk 2]
```diff
@@ -5,7 +5,17 @@
...
+
+* [source](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-source-release.zip) | [asc](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-source-release.zip.asc) | [sha512](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-source-release.zip.sha512)
+* [binary](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-bin-release.zip) | [asc](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-bin-release.zip.asc) | [sha512](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.7.0/apache-dubbo-incubating-2.7.0-bin-release.zip.sha512)
+
+### 2.6.5 
+
+* [source](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.6.5/apache-dubbo-incubating-2.6.5-source-release.zip) | [asc](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.6.5/apache-dubbo-incubating-2.6.5-source-release.zip.asc) | [sha512](https://dist.apache.org/repos/dist/dev/incubator/dubbo/2.6.5/apache-dubbo-incubating-2.6.5-source-release.zip.sha512)
```
Comment (ralf0131) [inline blog/en-us/download.md]: Hi, please update all the download links to https://dist.apache.org/repos/dist/release/incubator/dubbo/2.6.5/apache-dubbo-incubating-2.6.5-source-release.zip, note that from `dev` to `release`

**Comment by ralf0131** [comment]
Hi, 

I am sorry that I just updated the website with latest download link.
I am going to close this issue.

**Pull request closed without merging**